src/lib/crypto/bn.cpp

```diff
@@ -27,10 +27,9 @@
 #include "bn.h"
 #include <botan/ffi.h>
 #include <stdlib.h>
+#include <assert.h>
```

Comment:
I think technically these C headers are deprecated and we should be using `cassert`, but I know we're doing this all over the codebase and it's maybe not a concern right now

Reply:
You're right, just sometimes hard to think C++ when surrounded by the C code :)